src/test/42-function.ts

```diff
@@ -194,16 +198,24 @@ describe("validating function endpoints", function() {
           limit: 1,
         });
       } catch (err) {
-        expect(err.code).to.equal(500);
-        expect(err).to.have.property("message", "Internal Server Error");
+        let message;
+        if (err instanceof Error) {
+          expect(err).to.have.property("code", 500);
+          expect(err).to.have.property("message", "Internal Server Error");
+        } else message = String(err);
+        console.log(message);
       }
     });
     it("function.invokeFunctionWorkerWithoutQuery", async () => {
       try {
         await c8Client.function.invokeFunctionWorker("testSdkEv");
       } catch (err) {
-        expect(err.code).to.equal(400);
-        expect(err).to.have.property("message", "query is empty");
+        let message;
+        if (err instanceof Error) {
+          expect(err).to.have.property("code", 400);
+          expect(err).to.have.property("message", "query is empty");
+        } else message = String(err);
+        console.log(message);
```

Comment:
What is the purpose of storing the value in the message instead of writing the expected validation?

Reply:
We have expected validation in `if (err instanceof Error){ ... }` right? Since error type is `unknown` I just wanted to log anything that can pop up and satisfy TypeScript strict type. Do you have any other suggestion about how we can tackle this issue? @VikasPatel

Reply:
same as above